tests/bcl-test/templates/common/TestRunner.xUnit/XUnitTestRunner.cs

```diff
@@ -934,6 +934,7 @@ async Task<XElement> Run (Assembly assembly, string assemblyPath)
 					ITestFrameworkExecutionOptions executionOptions = GetFrameworkOptionsForExecution (configuration);
 					executionOptions.SetDisableParallelization (!RunInParallel);
 					executionOptions.SetSynchronousMessageReporting (true);
+					executionOptions.SetMaxParallelThreads (Environment.ProcessorCount / 2); // lets not be greedy and let the UI breath.
```

Comment:
Q. - Is there a C# equivalent to python's `//`? Does that make sense here?

Reply:
So // is floordiv in python I believe. I don't believe it is built in beyond:

`(long)(Math.Floor((double) a / b))` or whatever.

Reply:
And is not needed in this case :)